src/main.rs

```diff
@@ -19,141 +22,203 @@ fn main() -> Result<(), Box<std::error::Error>> {
     let namespace = env::var("TEST_PROJECT")?;
 
     // Friendly and polite greeting...
-    println!("{}{}{}",
-           "\n      Welcome to ShelfLife     \n",
-             "******We nuke old projects******\n",
-             " Get a job or get D E L E T E D \n");
+    println!(
+        "{}{}{}",
+        "\n      Welcome to ShelfLife     \n",
+        "******We nuke old projects******\n",
+        " Get a job or get D E L E T E D \n"
+    );
 
     let args: Vec<String> = env::args().collect();
-    if args.iter().any(|x| x == "k"){ let _query = query_known_namespace(token, endpoint, namespace); }
```

Comment:
No oneline if statements? :(

Reply:
That's `rustfmt`'s doing, not mine :P

Reply:
Whelp, if it's what the Rust gods desire, then I suppose I shall abide (at least 'till I know what I'm doing).